igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/model/GenericBuild.groovy

```diff
@@ -16,23 +16,37 @@
 
 package com.netflix.spinnaker.igor.build.model
 
+import com.fasterxml.jackson.annotation.JsonInclude
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
 class GenericBuild {
     boolean building
     String fullDisplayName
     int number
-    int duration
+    Integer duration
```

Comment:
Its not something I thought strongly about. It was just to make it work and I didn't see it when I cleaned up, I will change it back.